### ISSUE TITLE
Unable to sign in

### ISSUE THREAD

**Issue by ganesh3s3** [firebase/friendlyeats-android#6]
Android beginner trying to learn FIrestore here.

When trying to run this app with all the dependencies updated to the latest available version, I am unable to to sign in. I enter my email and click on next but the home screen is only shown for a second and it immediately switches back to the Sign in screen asking me to enter my email. The tutorial says I should be able to sign in and see the home screen without changing any code. Please forgive me if this issue is because I did something wrong. Thanks!

P.S. I have tried running the app as is, with all the included outdated dependencies and I still face the same issue.

**Comment by samtstern** [comment]
@ganesh3s3 are there any errors in the logs?  Do you have email authenticaiton enabled in the Firebase console?

**Comment by ganesh3s3** [comment]
Ty for your reply. I managed to fix the problem myself. The issue was I had other apps in the project and I used my personal email to sign-in in those. Since I gave this app the same email it was unable to sign me in since that email address was already in the list of users. 

I was really confused since neither the app nor the logcat gave me enough information about what went wrong. Now everything is clear.
